packages/ragbits-document-search/src/ragbits/document_search/ingestion/document_processor.py

```diff
@@ -49,7 +50,7 @@ def from_config(cls, providers_config: Optional[ProvidersConfig] = None) -> "Doc
         {
             DocumentType.TXT: YourCustomProviderClass(),
             DocumentType.PDF: UnstructuredProvider(),
-        }
+        }.
```

Comment:
This seems unusual - dot added to a line of example code. Maybe the code should be in a code block to avoid this?